Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -66,6 +81,23 @@ public override IEnumerable<ISetting> GetSettings()
             yield return stringTemplateSetting;
         }
 
+        private void QueryHelperLink_Click(object sender, EventArgs e)
+        {
+            if (string.IsNullOrWhiteSpace(urlSettings.CustomControl.Text))
+            {
+                MessageBox.Show(null, "Please fill the jira url to open the query helper", "Jira Url not filled", MessageBoxButtons.OK, MessageBoxIcon.Warning);
```

Comment:
* Inconsistencies: "jira" -> "Jira", "Url" -> "url"

Proposing to tweak the messages a little:
* "Jira Url not filled" -> "Invalid Jira url"
* "Please fill the jira url to open the query helper" -> "A valid url is required to launch the Jira query helper"

Thoughts?